design/object-metadata-s3.md

```diff
@@ -0,0 +1,48 @@
+# Object Metadata and S3
+
+### Overview
+
+lakeFS keeps the following metadata per object:
+
+1. Physical address
+1. Last modified time
+1. Size
+1. ETag
+1. Metadata
+1. Address type (relative / full)
+
+### What is missing?
+
+Mapped two issues that we currently do not address when we work with our S3 interface:
+
+1. Content-Type - Not keeping the posted object content-type and do not set it back when we get the object.
+    [Support Content-Type](https://github.com/treeverse/lakeFS/issues/2296)
+    We saw this issue when one of the clients used the value of the content type to identify a directory marker. But this is one of the basic properties that evey object store manages.
```

Comment:
Consider linking to the [S3 documentation on object metadata](https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingMetadata.html#SysMetadata) to show that this is (currently) the only piece of user-modifiable system metadata.